next.config.js

```diff
@@ -5,18 +5,29 @@ const nextConfig = {
   env: {
     NEXT_PUBLIC_VERCEL_URL: process.env.NEXT_PUBLIC_VERCEL_URL,
   },
-  source: "/api/:path*",
-  headers: [
-    { key: "Access-Control-Allow-Credentials", value: "true" },
-    { key: "Access-Control-Allow-Origin", value: "*" },
-    {
-      key: "Access-Control-Allow-Methods",
-      value: "GET,OPTIONS,PATCH,DELETE,POST,PUT",
-    },
-    {
-      key: "Access-Control-Allow-Headers",
-      value:
-        "X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version",
-    },
-  ],
+  images: {
+    domains: ["onlineweb4-prod.s3.eu-north-1.amazonaws.com"],
+  },
+  async headers() {
+    return [
+      {
+        source: "/api/:path*",
+        headers: [
+          { key: "Access-Control-Allow-Credentials", value: "true" },
+          { key: "Access-Control-Allow-Origin", value: "*" },
+          {
+            key: "Access-Control-Allow-Methods",
+            value: "GET,OPTIONS,PATCH,DELETE,POST,PUT",
+          },
+          {
+            key: "Access-Control-Allow-Headers",
+            value:
+              "X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version",
+          },
+        ],
+      },
+    ];
+  },
 };
```

Comment:
hva er dette

Reply:
For å bruke Image komponenten på fremmede linker, må man legge til tillatte domener